src/AcmTextInput/AcmTextInput.tsx

```diff
@@ -19,6 +24,9 @@ export function AcmTextInput(props: AcmTextInputProps) {
                 error = 'Required'
```

Comment:
Hard-coded English here

Reply:
Agreed. Need to discuss a good approach to this.